examples/auth/oauth2/keycloak/keycloak.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set -o errexit  # Used to exit upon error, avoiding cascading errors
```

Comment:
are we using keycloak here?

Reply:
I'll remove the stale reference. Sorry about that. I was going to experiment with Keycloak at somepoint.